Update gradle in gitpod/workspace-full

**Describe the bug**
Currently running `gradle init` in a fresh `gitpod/workspace-full` based workspace yields:
```
FAILURE: Build failed with an exception.

* What went wrong:
java.lang.ExceptionInInitializerError (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

**To Reproduce**
 1. Open a fresh repository
 1. run `gradle init --type java-library`

**Expected behavior**
Gradle should initialize the project.


It seems to be a [gradle incompatibility](https://stackoverflow.com/questions/42851099/java-lang-exceptionininitializererror-with-gradle) with java 8 (we have `jdk 1.8.0_201`). Updating to `gradle-4.10.3` solves the issues; sadly this is not available from the ubuntu repos.

I am having a very similar if not the same issue.
Steps to reproduce:
 - Start Theia with `yarn --cwd ./examples/browser start --hostname 0.0.0.0 --port 3000`
 - Open a terminal and create a new folder: `mkdir jdt-ls-gh-28 && cd jdt-ls-gh-28`.
 - Execute `gradle init --type java-library`.

```
Picked up JAVA_TOOL_OPTIONS: -Xmx3189m
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Unable to start the daemon process.
This problem might be caused by incorrect configuration of the daemon.
For example, an unrecognized jvm option is used.
Please refer to the user guide chapter on the daemon at https://docs.gradle.org/4.4.1/userguide/gradle_daemon.html
Please read the following process output to find out more:
-----------------------
Picked up JAVA_TOOL_OPTIONS: -Xmx3189m

FAILURE: Build failed with an exception.

* What went wrong:
java.lang.ExceptionInInitializerError (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/.werft/build.ts
+++ b/.werft/build.ts
@@ -548,7 +548,9 @@ export async function deployToDev(deploymentConfig: DeploymentConfig, workspaceF
         } else if (!!deploymentConfig.analytics) {
             flags += ` --set analytics.writer=${deploymentConfig.analytics!}`;
         }
-
+        if (deploymentConfig.withObservability) {
+            flags += ` -f ../.werft/values.tracing.yaml`;
+        }
         werft.log("helm", "extracting versions");
         try {
             exec(`docker run --rm eu.gcr.io/gitpod-core-dev/build/versions:${version} cat /versions.yaml | tee versions.yaml`);

--- a/.werft/values.dev.yaml
+++ b/.werft/values.dev.yaml
@@ -12,10 +12,6 @@ version: not-set
 imagePullPolicy: Always
 
 authProviders: []
-tracing:
-  endpoint: http://otel-collector:14268/api/traces
-  samplerType: const
-  samplerParam: "1"
 
 # we hit the "max. 110 pods/node" situation pretty often with our current core-dev setup.
 # the proper way to fix those would be to adjust the CIDR for workload NodePool which is a bit more work.

--- a/.werft/values.tracing.yaml
+++ b/.werft/values.tracing.yaml
@@ -1,0 +1,4 @@
+tracing:
+  endpoint: http://otel-collector:14268/api/traces
+  samplerType: const
+  samplerParam: "1"

